sql/migrations/005_resolution_creation_timestamp.sql

```diff
@@ -0,0 +1,8 @@
+-- +migrate Up
+
+-- Adds the resolution creation timestamp on resolution table
+ALTER TABLE "resolution" ADD COLUMN "created" TIMESTAMP with time zone DEFAULT now() NOT NULL;
```

Comment:
Wonder if the migration process should not manage differently already created/already finished task.
If we have 200 tasks in the database, with state DONE, at the time we run the `created` value will have a value after `last_start` and `last_stop`.
Should we put the value to `last_start` for existing task?

Making the query to something like
```sql
ALTER TABLE "resolution" ADD COLUMN "created" TIMESTAMP with time zone;
UPDATE "resolution" SET created = last_start;
ALTER TABLE "resolution" ALTER COLUMN "created" SET NOT NULL;
```

@loopfz @wI2L thoughts?
```

Reply:
@loopfz @wI2L  up

Reply:
last_stop sounds better? but yeah, agree

Reply:
I agree. Since the `created` field represents the resolution creation, shouldn't `last_start` be closer to it ?